Configure

```diff
@@ -919,8 +919,17 @@ $target{dso_extension}=$target{shared_extension_simple};
 $config{cross_compile_prefix} = $ENV{'CROSS_COMPILE'}
     if $config{cross_compile_prefix} eq "";
 
+# This is textbook from the perlvar manual.  It makes sure we have
+# a perl path that can be used as a file name, or as the case may be,
+# as something that can be used with the hashbang script syntax.
+use Config;
+my $secure_perl_path = $Config{perlpath};
```

Comment:
Or oh, were you saying that I could be a bit less paranoid and choose the first of the two textbook examples shown above?  That's a good point...